actors/builtin/miner/policy.go

```diff
@@ -47,3 +47,9 @@ func precommitDeposit(sectorSize abi.SectorSize, duration abi.ChainEpoch) abi.To
 func temporaryFaultFee(weights []*power.SectorStorageWeightDesc, duration abi.ChainEpoch) abi.TokenAmount {
 	return big.Zero() // PARAM_FINISH
 }
+
+// MaxFaultsCount is the maximum number of faults that can be declared
+const MaxFaultsCount = 32 << 20
+
+// ProvingPeriod defines the frequency of PoSt challenges that a miner will have to respond to
+const ProvingPeriod = 300
```

Comment:
please remove `SurprisePoStPeriod` from the `power/policy.go` file accordingly. Fwiw I think `fallbackPoStPeriod` or something is better. Proving is too confusing/overloaded